src/Processors/Transforms/Streaming/GlobalAggregatingTransform.h

```diff
@@ -28,6 +28,8 @@ class GlobalAggregatingTransform final : public AggregatingTransform
     bool prepareFinalization(Int64 min_watermark) override;
 
     void finalize(const ChunkContextPtr & chunk_ctx) override;
+
+    static constexpr VersionType V2 = 3;
```

Comment:
Let's start commenting / documenting the version changelogs which are important.